packages/frontend/src/components/MkNoteDetailed.vue

```diff
@@ -392,29 +395,26 @@ if (appearNote.value.reactionAcceptance === 'likeOnly') {
 	});
 }
 
-function renote(viaKeyboard = false) {
+async function renote() {
 	pleaseLogin();
 	showMovedDialog();
 
-	const { menu } = getRenoteMenu({ note: note.value, renoteButton });
-	os.popupMenu(menu, renoteButton.value, {
-		viaKeyboard,
-	});
+	const { menu } = await getRenoteMenu({ note: note.value, renoteButton, canRenote: canRenote.value });
```

Comment:
```suggestion
	const { menu } = getRenoteMenu({ note: note.value, renoteButton, canRenote: canRenote.value });
```
getRenoteMenuが非同期関数なのはtaiy fork固有なので、ここは戻して良いかも